Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM rust:1.39.0-buster as builder
+FROM rust:1.40.0-buster as builder
```

Comment:
Happy to back out this change since it's a bit unrelated, but figured I'd just grab it while I was in here.